pkgs/by-name/st/stats/package.nix

```diff
@@ -1,29 +1,44 @@
 {
   lib,
-  stdenvNoCC,
-  fetchurl,
-  undmg,
+  stdenv,
+  fetchFromGitHub,
   nix-update-script,
+  apple-sdk,
+  swift,
+  xcbuildHook,
 }:
 
-stdenvNoCC.mkDerivation (finalAttrs: {
+stdenv.mkDerivation (finalAttrs: {
```

Comment:
You need to use `swiftPackages.stdenv` when using Swift; the build on `x86_64-darwin` is broken right now.